### PR TITLE
fix windows issue caching schemas; create a cookbook to develop on wi…

### LINKED ISSUE #197: Is the service loading JSON schemas when running on Windows?
As oneview-redfish-toolkit user, I should be able to run the service with local JSON schema validation enabled for all resources available.

### PR DESCRIPTION
**Pull request by ffdarkpenguin**
closes #197

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -26,7 +26,7 @@ commands = {posargs}
 commands = python setup.py test --coverage --testr-args='{posargs}'
 
 [testenv:docs]
-basepython=python3
+basepython=python
```
Comment (ricardoas) [inline tox.ini]: nice!

File: doc/windows-dev.txt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+2.  Install python 3.5+ from python.org
+3.  clone the project:
+    git clone https://waffle.io/HewlettPackard/oneview-redfish-toolkit
+4.  Access project directory:
+    cd oneview-redfish-toolkit
+5.  Create a virtual environment(venv):
+    py -m venv .venv
+    NOTE: .venv is the directory of the virtual environment. You can name it whatever you like, given it's not an existing dir in the project.
+6.  activate the virtual env:
+    ./venv/scripts/activate
```
Comment (ricardoas) [inline doc/windows-dev.txt]: I tried this here but it only worked when using: `.\.venv\Scripts\activate.bat`
Any idea if this is specific to my windows 7 environment?

Comment (ffdarkpenguin) [reply doc/windows-dev.txt]: the activate script checks if you are running command or powershell and calls activate.bat or activate.ps1 accordingly.
I've tested with windows 10 in powershell and command prompt and worked. I you want I can build a windows 7 environment. But I think it's easier just to explain and tell people about the other activate scripts. Check if after the update and tell me what you want...

File: tox.ini (hunk 1 above)
Comment (ffdarkpenguin) [reply tox.ini]: ty

**Pull request merged**